tests/test_views.py

```diff
@@ -2,6 +2,8 @@
 
 from flasql import views
 
+from tests.fixtures import app # noqa
```

Comment:
Ah, right. I think you could do `from tests import fixtures`, as long as they're loaded they're available.